### PR TITLE
Created an utility component that simplifies the process of rendering images

### PR DESCRIPTION
**Pull request by Giovym**
Created the Image utility component with the following criteria:

✅ The  component should accept the image source URL as a prop and render the corresponding image.
✅ It should handle lazy loading of images to improve performance.
✅ Provide optional props to specify alt text, fallback images, and other image-related attributes.

### CODE REVIEW

File: lib/utils/Image.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const Image = ({ src, loading, alt, fallback, style }) => {
```
Comment (DavidHDev) [inline lib/utils/Image.jsx:3]: Instead of using default exports please just export it here
"export const Image..."

File: lib/utils/Image.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  const handleBrokenImage = (event) => (event.target.src = fallback);
+
+  return (
+    <img
+      src={src}
```
Comment (DavidHDev) [inline lib/utils/Image.jsx:8]: Maybe we should also add these props: width, height, className (could come in handy)

**Comment by DavidHDev** [comment]
@Giovym please also edit the `index.js` and `index.d.ts` files with the required changes for this feature.

File: lib/utils/Image.jsx (hunk 2 above)
Comment (Giovym) [reply lib/utils/Image.jsx:17]: Yes, I will

**Review by Giovym — COMMENTED** [review]
All the suggestions have been implemented.

File: lib/utils/Image.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      loading={loading}
+      alt={alt}
+      onError={handleBrokenImage}
+      style={style}
+      width={width}
+      height={height}
+      className={className}
```
Comment (m10rten) [inline lib/utils/Image.jsx:24]: Although @DavidHDev suggested to add these props, you could easily add all img props with a `...props` and the right typing.

Comment (Giovym) [reply lib/utils/Image.jsx]: Thanks for your suggestion. I implemented your advice. I hope now the component is correct!

File: lib/index.d.ts [hunk 4]
```diff
@@ -194,3 +194,14 @@ export declare const Show: FC<{
...
+export declare const Image: FC<{
+  src: string;
+  loading?: "eager" | "lazy";
+  alt?: string;
+  fallback?: string;
+  style?: React.CSSProperties;
+  width?: number;
+  height?: number;
+  className?: string;
+}>;
```
Comment (m10rten) [inline lib/index.d.ts:207]: You did the ...props correctly, but you can add the React variant of the img tag here, so Image extends from img tag.

Comment (Giovym) [reply lib/index.d.ts:207]: Do you mean something like this: [key:string]: any;

**Review by DavidHDev — APPROVED** [review]

**Pull request merged**